public/configs/cctp.json

```diff
@@ -4,12 +4,6 @@
     "tokenAddress": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
     "name": "Ethereum"
   },
-  {
```

Comment:
i can add this back if anyone cares, i think for now we're better off without it though

Reply:
I think we can replace this with sepolia, so that in testnet the deposit form isn't too broken with no assets:

sepolia usdc address is: `0x1c7d4b196cb0c7b01d743fbc6116a902379c7238`

<img width="454" alt="Screenshot 2024-07-08 at 3 00 49 PM" src="https://github.com/dydxprotocol/v4-web/assets/1062252/15eb589e-b5ba-4307-9b00-1a9d2861f225">

Reply:
ah you're totally right, good idea